File: seed/csharp-sdk/server-sent-events-openapi/src/SeedApi.DynamicSnippets/Example30.cs
using SeedApi;

namespace Usage;

public class Example30
{
    public async Task Do() {
        var client = new SeedApiClient(
            clientOptions: new ClientOptions {
                BaseUrl = "https://api.fern.com"
            }
        );

        await foreach (var item in client.StreamXFernStreamingNullableConditionStreamAsync(
            new StreamXFernStreamingNullableConditionStreamRequest {
                Query = "query",
                Stream = true
            }
        ))
        {
            /* consume each item */
        }
        ;
    }

}

File: seed/java-sdk/server-sent-events-openapi/with-wire-tests/src/main/java/com/snippets/Example30.java
package com.snippets;

import com.seed.api.SeedApiClient;
import com.seed.api.requests.StreamXFernStreamingNullableConditionStreamRequest;

public class Example30 {
    public static void main(String[] args) {
        SeedApiClient client =
                SeedApiClient.builder().url("https://api.fern.com").build();

        client.streamXFernStreamingNullableConditionStream(StreamXFernStreamingNullableConditionStreamRequest.builder()
                .query("query")
                .build());
    }
}
